definitions/variable/emissions/emissions.yaml

```diff
@@ -43,52 +71,113 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Biomass Burning:
+    description: Emissions from agricultural waste burning
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land:
     description: Emissions of {Level-3 Species} from forestry and other land use
     unit: "{Level-3 Species}"
+    components:
+      - Emissions|{Level-3 Species}|AFOLU|Land|Forest Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Cropland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Grassland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands
+      - Emissions|{Level-3 Species}|AFOLU|Land|Settlements
+      - Emissions|{Level-3 Species}|AFOLU|Land|Other Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Fires
+- Emissions|{Level-3 Species}|AFOLU|Land|Forest Land:
+    description: Emissions and removals from lands with woody vegetation
```

Comment:
Does this include CO2 emission from deforestation? If so, pleae add it to the desc.